python/mleap/sklearn/extensions/data.py

```diff
@@ -109,18 +119,19 @@ def predict(self, X):
 This is because the Imputer both in Spark and MLeap operates on a scalar value and if we were to add a feature extractor in
 front of it, then it would serialize as operating on a tensor and thus, fail at scoring time. 
 """
-class Imputer(SklearnImputer):
+class Imputer(SKLearnImputer):
 
-    def __init__(self, missing_values="NaN", strategy="mean",
-                 axis=0, verbose=0, copy=True, input_features=None, output_features=None):
+    def __init__(self, missing_values=np.nan, strategy="mean",
+                 fill_value=None, verbose=0, copy=True, add_indicator=False,
```

Comment:
should we have the new kwarg, "add_indicator" be added at the end of the signature so folks using this constructor with positional args don't break